### DISCUSSION TITLE
How to search multiple types?

### DISCUSSION THREAD

**Discussion by graveship** [JohnnyCrazy/SpotifyAPI-NET#905] [Q&A] [answered]
Im using Spotify.Search.Item and in the documentation it states that you can search using a comma seperated list.
`
 type: A comma-separated list of item types to search across. Valid types are: album , artist, playlist, track, show and episode. Search results include hits from all the specified item types.
`

The constructor paramter is a single type public SearchRequest(Types type, ...)  and passing in a string or list of types is not allowed/supported?. I want to specifcally search for only albums, tracks and artists any other data is not needed. I am currently using the All type and ignoring any of the results returned from audiobooks and episodes but i was wondering if there was a way to only search for multiple types instead of having the API return data that is never going to be used everytime?

**Comment by JohnnyCrazy** [comment, marked as answer]
Hi,

```csharp
var search = new SearchRequest(SearchRequest.Types.Album | SearchRequest.Types.Playlist, "Hello World");
```

This should do the trick.

**Comment by graveship** [comment]
Thanks didn't realise this was possible.
